packages/isl/package.desc

```diff
@@ -1,6 +1,6 @@
 repository='git git://repo.or.cz/isl.git'
 bootstrap='./autogen.sh'
-mirrors='http://isl.gforge.inria.fr'
+mirrors='https://libisl.sourceforge.io'
```

Comment:
This part is fixed upstream in commit cfb7d07ae1e0ef4dbf14f40a5744d1abd382d000

Reply:
@lgirdwood I cherry-picked the upstream change to package.desc.  Can I resolve this conversation?